synapse/tools/storm.py

```diff
@@ -349,8 +349,7 @@ async def _get_tag_completions(self, prefix='', limit=100):
         +:depth<=$depth
         | uniq
         | limit $limit
-        | $leaf = $lib.true { -> syn:tag:up | limit 1 | $leaf = $lib.false }
-        $doc = ''
+        | $doc = ''
         if $node.props.doc {
```

Comment:
why did you add a pipe here?

Reply:
The previous line was removed (the one with `$leaf = ...`) because it wasn't being used.

Reply:
This is after a `limit` command. The `$leaf` value is unused.